CompuCell3D/core/Demos/PluginDemos/FocalPointPlasticityLinks/Simulation/FocalPointPlasticityLinksSteppables.py

```diff
@@ -0,0 +1,93 @@
+from cc3d.core.PySteppables import *
+
+import random
+
+
+class FocalPointPlasticityLinksSteppable(SteppableBasePy):
+
+    def __init__(self, frequency=1):
+
+        SteppableBasePy.__init__(self, frequency)
+
+    def step(self, mcs):
```

Comment:
I think this code looks much nicer than any previos code that involved links

Reply:
Nice!